src/components/modals/WalletListMenuModal.js

```diff
@@ -83,16 +83,33 @@ class WalletListMenuModalComponent extends PureComponent<Props> {
       const { currencyCodes, label, value } = option
       if (currencyCodes != null && !currencyCodes.includes(currencyCode)) continue
 
+      if (option.value === 'split' && this.checkSplitExclude()) continue
+
       const temp = { label, value }
+
       if (option.value === 'split') {
         const splitString = s.strings.string_split_wallet
         const currencyName = currencyCode === 'BTC' ? 'Bitcoin Cash' : 'Bitcoin SV'
         temp.label = sprintf(splitString, currencyName)
       }
+
       this.options.push(temp)
     }
   }
 
+  checkSplitExclude() {
+    const { walletName } = this.props
+
+    if (!walletName) return false
+
+    return (
+      EXCLUDE_SPLIT_WALLETS.findIndex((item: string) => {
+        const regex = new RegExp(item, 'i')
```

Comment:
Use "indexOf" or "includes" but not a regex.
Check out the performance differences, its massive:

<img width="1017" alt="Screen Shot 2021-06-08 at 20 34 16" src="https://user-images.githubusercontent.com/4012041/121289324-99c04c00-c899-11eb-81bf-4c15cd0c5d77.png">

Reply:
The index of method is not insensitive if I accidentally set segwit to EXCLUDE_SPLIT_WALLETS = ['segwit'] because the lowercase but wallet name is “Btc (Segwit)” . IndexOf method doesn't working for me. We can added individual param (isSegwit) to wallet for it if you diсide that it is actual.